includes/Modules/Analytics.php

```diff
@@ -119,6 +119,11 @@ public function register() {
 	 * @return bool
 	 */
 	protected function is_tracking_disabled() {
+		$settings = $this->get_settings()->get();
+		if ( ! apply_filters( 'googlesitekit_allow_tracking_disabled', $settings['useSnippet'] ) ) {
```

Comment:
It's good to mention that a filter is documented elsewhere, so something like this should do the trick:

```suggestion
		// This filter is documented below, later in this function.
		if ( ! apply_filters( 'googlesitekit_allow_tracking_disabled', $settings['useSnippet'] ) ) {
```

Reply:
You mean direct the reader to `Tag_Manager:: filter_analytics_allow_tracking_disabled`?

Reply:
Yup! 🙂  Sorry, I thought it was in this file, but yes, pointing to there is good.